django/contrib/auth/mixins.py

```diff
@@ -2,7 +2,6 @@
 
 from django.conf import settings
 from django.contrib.auth import REDIRECT_FIELD_NAME
-from django.contrib.auth.views import redirect_to_login
```

Comment:
@sarahboyce I guess we no longer need an inner import :D

Reply:
Star thank you!